### ISSUE TITLE
Idled out of game while researching

### ISSUE THREAD

**Issue by rpherbig** [rpherbig/dr-scripts#302]
```

[crossing-training]>research UTILITY 300

You confidently begin to bend the mana streams in the air in front of you, testing the limits of your ability to manipulate the world with utility spells.

[crossing-training: ***Skills capped and researching***]

You continue making theoretical strides.
You continue to study Utility magic.
You continue to study Utility magic.

[crossing-training]>tdp

You have 35 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue to study Utility magic.
You continue making theoretical strides.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue making theoretical strides.
You continue to study Utility magic.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue making theoretical strides.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue to study Utility magic.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue to study Utility magic.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue making theoretical strides.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You make definite progress in your project about Utility Patterns Research and decide to take a break.  However, there is still more to learn before you arrive at a breakthrough.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training]>research UTILITY 300

You confidently begin to bend the mana streams in the air in front of you, testing the limits of your ability to manipulate the world with utility spells.

[crossing-training: ***Skills capped and researching***]

You continue making theoretical strides.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue to study Utility magic.
You continue to study Utility magic.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

The symbols and sigils around you seem to pulse with meaning as you continue your project.
You continue to study Utility magic.
You continue to study Utility magic.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue to study Utility magic.
You continue to study Utility magic.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

The symbols and sigils around you seem to pulse with meaning as you continue your project.
You continue making theoretical strides.

[crossing-training]>tdp

You have 36 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue to study Utility magic.

[crossing-training]>tdp

You have 37 TDPs.

[crossing-training: ***Skills capped and researching***]

Noting that your research is progressing nicely, you continue to focus on it.
The symbols and sigils around you seem to pulse with meaning as you continue your project.
You continue to study Utility magic.

[crossing-training]>tdp

You have 37 TDPs.

[crossing-training: ***Skills capped and researching***]

You continue to study Utility magic.
You continue making theoretical strides.

[crossing-training]>tdp

You have 37 TDPs.

You continue to study Utility magic.
You continue to study Utility magic.
Noting that your research is progressing nicely, you continue to focus on it.
You continue to study Utility magic.
You make definite progress in your project about Utility Patterns Research and decide to take a break.  However, there is still more to learn before you arrive at a breakthrough.
```

**Comment by rcuhljr** [comment]
I'm assuming your ruby hung or something.

**Comment by rpherbig** [comment]
Haven't seen this since. Possibly fixed by #320?
